files/address.csv

```diff
@@ -0,0 +1,3 @@
+0|centro,guadalajara,jalisco|20.6866131|-103.3507872
+1|wizeline, zapopan, jalisco, mexico|20.6443271|-103.4163436
+2|la minerva, guadalajara, jalisco, mexico|20.6743943|-103.3874128
```

Comment:
All the CSV files should finish with an empty line.
If your app adds a new row, it will be added where the cursor is, so you will have two registers in the same line.